pkg/client/client.go

```diff
@@ -500,4 +503,18 @@ func FormatStatusResponse(w io.Writer, sr *models.StatusResponse, sd StatusDetai
 
 		fmt.Fprintf(w, "Hubble:\t%s\n", strings.Join(fields, "\t"))
 	}
+
+	if sd&StatusBPFMapSizeDetails != 0 && sr.BpfMaps != nil {
+		dynamicSizingStatus := "off"
+		ratio := sr.BpfMaps.DynamicSizeRatio
+		if 0.0 < ratio && ratio <= 1.0 {
+			dynamicSizingStatus = fmt.Sprintf("on (ratio: %f)", ratio)
+		}
+		fmt.Fprintf(w, "BPF Map sizes:\tdynamic sizing: %s\n", dynamicSizingStatus)
+		fmt.Fprintf(w, "  CT TCP\t%d\n", sr.BpfMaps.CtTCPMapSize)
+		fmt.Fprintf(w, "  CT any\t%d\n", sr.BpfMaps.CtAnyMapSize)
+		fmt.Fprintf(w, "  NAT\t%d\n", sr.BpfMaps.NatMapSize)
+		fmt.Fprintf(w, "  Neigh\t%d\n", sr.BpfMaps.NeighMapSize)
+		fmt.Fprintf(w, "  SockRev\t%d\n", sr.BpfMaps.SockRevMapSize)
```

Comment:
Nit: maybe we should include all maps managed by Cilium?

Reply:
Yes, makes sense.